timer.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Timer</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="background">
+    <div class="timers" id="timers">
+     <h1>Timer</h1>
+      <p id="count">0</p>
+        <button id="startBtn">Start</button>
+        <button id="pauseBtn">Pause</button>
+        <button id="resumeBtn">Resume</button>
+        <button id="stopBtn">Stop</button>
+    </div>
+    <script src="time.js"></script>
```

Comment:
Your Javascript filename does not match with your script src in html. It should be like:

<script src="timer.js"></script>